pkg/pool-manager/pool.go

```diff
@@ -176,3 +177,24 @@ func getNextMac(currentMac net.HardwareAddr) net.HardwareAddr {
 
 	return currentMac
 }
+
+// Checks if the namespace of an instance is opted in for kubemacpool
+func (p *PoolManager) IsInstanceOptedIn(namespaceName string, labelSelector string) (error, bool) {
+
+	isNamespaceOptedIn := false
```

Comment:
There is no logic relation between this line and the next one. Please insert an empty line